exercises/harriet_brown/ex02_oo_basics/src/getInput.cpp

```diff
@@ -0,0 +1,62 @@
+#include "../inc/getInput.h"
+
+
+void getInput(std::string question_text, int &ans)
```

Comment:
Good use of functions to break this up. In most cases I would opt for returning a value rather than using an output parameter so something like `int getInput(const std::string & question_text)`. It makes reading the code easier and doesn't require you to define a variable first to capture it when calling the function.